### PR TITLE
Create folder for data files in generator templates

### PR DESCRIPTION
**Pull request by pigeonfresh**
From experience I like to create multiple data files and store them in a folder called data. This PR will update the generator template used to scaffold a block.

### CODE REVIEW

**Comment by ThaNarie** [comment]
I don't really have an opinion on this, let's ask some others what they think :)

After that, this PR should be updated to move the default muban components/usage to reflect these changes, and of course the documentation :)

**Comment by melsd** [comment]
Same as you and Ronald, I usually create a data folder to store all the data files.

**Comment by larsvanbraam** [comment]
I'm fine with adding a data folder!

**Comment by pigeonfresh** [comment]
I've updated the boilerplate data to reflect the use of a folder for data

File: src/app/component/block/two-col/preset.js [hunk 1]
```diff
@@ -6,17 +6,17 @@ storiesOf('Two Col', require('./two-col'))
     'default',
     'A block that has two columns of text, each with its own title.',
     `<hbs>
-			{{> two-col @root }}
-		</hbs>`,
+      {{> two-col @root }}
+    </hbs>`,
     require('./data'),
```
Comment (ThaNarie) [inline src/app/component/block/two-col/preset.js]: forgot this one? :)

**Comment by pigeonfresh** [comment]
@ThaNarie Do we need anything else before this can be merged?

**Comment by ThaNarie** [comment]
Nope, just the one comment I just posted, after that you can merge it :)

**Pull request merged**